backend/lib/adapters/gateways/dxtr/orders.dxtr.gateway.ts

```diff
@@ -660,4 +579,75 @@ export class DxtrOrdersGateway implements OrdersGateway {
       return Promise.reject(new CamsError(MODULE_NAME, { message: queryResult.message }));
     }
   }
+
+  private setUpCommonParameters(txId: string, context: ApplicationContext<unknown>) {
+    const maxTxId: number = parseInt(txId);
+
+    // TODO: We need to consider whether we partially load cosmos by chapter. This has ongoing data handling concerns whether we load all or load partially.
+    const chapters: string[] = ['15'];
+    if (context.featureFlags['chapter-eleven-enabled']) chapters.push('11');
+    if (context.featureFlags['chapter-twelve-enabled']) chapters.push('12');
+
+    const regions = REGIONS_TO_SYNC;
+
+    const params: DbTableFieldSpec[] = [];
+    params.push({
+      name: 'txId',
+      type: mssql.BigInt,
+      value: txId,
+    });
```

Comment:
We need to diverge in that the txId for transfers and consolidations needs to be different.

This can't be in a common function anyway though. Oops.